pkg/virt-operator/resource/generate/components/deployments.go

```diff
@@ -261,7 +265,7 @@ func NewApiServerDeployment(namespace string, repository string, imagePrefix str
 	container := &deployment.Spec.Template.Spec.Containers[0]
 	container.Command = []string{
 		VirtAPIName,
-		"--port",
```

Comment:
I think this particular constant extraction doesn't have much value in it (maybe on the contrary), since all other arguments (`--console-server-port`, etc) are still inlined. I realize this is being reported as duplicate strings by SonarCloud, but I'd suggest to have an exception for this particular instance.

Reply:
100% agree. thanks